src/styles/components/header.scss

```diff
@@ -89,18 +89,14 @@ header.layout-new-header {
     }
   }
 
-  > a {
+  a {
     color: var(--logo-color-blue-white);
     cursor: pointer;
     text-decoration: none;
     transition: border-bottom 0.3s;
 
-    &.active {
+    &.active:not(.roar-home) {
       border-bottom: 2px $human-pink solid;
```

Comment:
The transition on active is a bit off. Not sure if it's fixable or not, but it looks like we're moving from regular font-weight to bold without a transition.